### DISCUSSION TITLE
[FEEDBACK]: apiPort setting is not available in devproxyrc.json

### DISCUSSION THREAD

**Discussion by cronologi-davukovi** [dotnet/dev-proxy#1218] [Q&A] [answered]
### Type of issue

Outdated article

### Feedback

The documentation mentions various proxy settings but does not include information about the apiPort setting in devproxyrc.json. It would be helpful to clarify if this setting is supported or intended to be configurable in devproxyrc.json. If not, please update the documentation to explicitly state its absence to avoid confusion.

Link to documentation: [Proxy Settings - Microsoft Learn](https://learn.microsoft.com/en-us/microsoft-cloud/dev/dev-proxy/technical-reference/proxy-settings)

Thank you for your attention!

### Page URL

https://learn.microsoft.com/en-us/microsoft-cloud/dev/dev-proxy/technical-reference/proxy-settings

### Content source URL

https://github.com/MicrosoftDocs/microsoft-cloud/blob/main/docs/dev/dev-proxy/technical-reference/proxy-settings.md

### Author

@garrytrinder

### Document Id

6004001f-4791-7ca2-9818-d788175f3f4b

**Comment by waldekmastykarz** [comment]
Thank you for reaching out. `apiPort` is the first setting mentioned in the doc:

![Image](https://github.com/user-attachments/assets/e2a80db8-6821-4bdf-85a1-1caac8f23bb6)

Because it's included in this listing, it means that it can be configured in devproxyrc.json. Or am I misunderstanding your comment and you mean something else? 🙂

**Comment by cronologi-davukovi** [comment]
The apiPort is not available on the schema of the config file https://github.com/microsoft/dev-proxy/blob/main/schemas/v0.23.0/rc.schema.json, and when I was trying to set the apiPort in the config, devproxy was not working for me. Given these circumstances, I was confused whether the documentation needs to be updated.

**Comment by waldekmastykarz** [comment]
Understood, thank you for clarifying. Let me verify it and I'll get back to you when I know more.

**Comment by waldekmastykarz** [comment]
apiPort missing in the schema is a bug and we'll fix it asap

**Comment by waldekmastykarz** [comment, marked as answer]
If I specify apiPort in Dev Proxy config, it seems to be working just fine:

![Image](https://github.com/user-attachments/assets/6e1c09e3-7d17-4e8c-9d01-f5258aabd8a4)

Could you please share with me your repro steps?

**Comment by waldekmastykarz** [comment]
Closing due to lack of response
